src/components/Button.js

```diff
@@ -6,7 +6,7 @@ const Button = styled.button`
   padding: 20px;
   margin: 20px;
   width: 332px;
-  background-color: #c3073f;
+  background-color: #00ff00;
   color: #1a1a1d;
```

Comment:
Maybe you could add GlobalStyles to your app, to have a better control on your styles.

Reply:
good point! will do. :)